README.md

```diff
@@ -47,13 +47,32 @@ Alternatively, you can configure Vault connection using next environmnt variable
 ## Preparing secrets in Vault
 
 You need to store your secrets with special naming rules.
-First of all, all secrets should use KV2 storage and have prefix `data/{app_alias}/`.
-For example, if your app has alias `sampleapp` and you want to have configuration option `ConnectionString` your secret path would be `data/sampleapp/ConnectionString`.
+First of all, all secrets should use KV2 storage and have prefix `{app_alias}/{env}`.
+For example, if your app has alias `sampleapp` and environment `producton` and you want to have configuration option `ConnectionString` your secret path would be `sampleapp/producton`.
```

Comment:
What do you think if we reverse the path to be `{env}/{app_alias}`? That would simplify creating Vault policies by env name.

Reply:
This description is purely advisory in nature, and everyone can do whatever they want. If this is important to you, you can reflect this point in the Readme.